### PR TITLE
Added feature to add different profiles to switch between

### PR DESCRIPTION
**Pull request by PaiAkshay998**
- By passing the load parameter and the config you want to load in the command line, you can load a previously used configuration without having to re-enter it over and over again.
```bash
./main.sh load sample
```
- Extra option to save while entering a new configuration

### CODE REVIEW

File: main.sh [hunk 1]
```diff
@@ -55,68 +55,107 @@ if [ "$1" = "list" ]; then
...
+use_same="n"
+https_host=""
+https_port=""
+ftp_host=""
+ftp_port=""
+username=""
+password=""
+save_for_reuse=""
+profile_name=""
+use_auth="n"
```
Comment (himanshub16) [inline main.sh]: Would recommend to place `use_auth` above `username` and `password`. Variables of same type should be in prioximity to each other.

File: main.sh [hunk 2]
```diff
@@ -55,68 +55,107 @@ if [ "$1" = "list" ]; then
...
+    =========\e[0m
+    Tool to set up system wide proxy settings on Linux.
+    \e[2m\e[33m🌟\e[0m\e[3m Star it \e[0m : \e[4m\e[34m https://github.com/himanshub16/ProxyMan \e[0m
+
+    \e[4mThe following options are available : \e[0m
+    \e[1m set \e[0m    : \e[2m Set proxy settings \e[0m
+    \e[1m unset \e[0m  : \e[2m Unset proxy settings \e[0m
+    \e[1m list \e[0m   : \e[2m List current settings \e[0m
+
+    "
```
Comment (himanshub16) [inline main.sh]: These lines are indented, and thus appear the same when the script runs. They should be aligned to the left.

**Comment by himanshub16** [comment]
Hi @PaiAkshay998 
You made a decent contribution and I really acknowledge that. 
I have made a couple of remarks to be taken care of.

However, I have some recommendations, that'd make your implementation of configs even better.

* Save the config file in `key=value` format. This makes things more human readable and easier if someone wants to view and find play with them. The values can be extracted using combination of `grep` and `cut` as:

_sample_file.conf_
```
http_proxy=http://ip.domain:port
```

Required code:
```
http_proxy=`grep http_proxy -i sample_file.conf  | cut -d= -f2`
```

* There should be a way to use `load` after starting the script, same way `list` works after starting as well as when sent as parameter.

File: main.sh [hunk 3]
```diff
@@ -55,68 +55,107 @@ if [ "$1" = "list" ]; then
...
+        else
+            echo -ne "\e[36m HTTPS Proxy host \e[0m " ; read https_host
+            echo -ne "\e[32m HTTPS Proxy port \e[0m " ; read https_port
+            echo -ne "\e[36m FTP   Proxy host \e[0m " ; read ftp_host
+            echo -ne "\e[32m FTP   Proxy port \e[0m " ; read ftp_port
+        fi
     fi
 fi
 
+if [ "$1" = "load" ]; then
```
Comment (himanshub16) [inline main.sh]: This `if` can be placed just above the previous one, which will leave it in proximity to the place where code related to `load` is there. 
Always remember, statements doing the same task should be closer to each other. This helps a lot when the size increases and when debugging.

Moreover, one of the two `if` can be replaced by an `else`. That'd be more elegant when the `if [ $1 == "load" ]; then` is in `if` and the other one as `else`.

File: main.sh (hunk 1 above)
Comment (PaiAkshay998) [reply main.sh]: 👍

File: main.sh (hunk 2 above)
Comment (PaiAkshay998) [reply main.sh]: Done 👍

**Comment by himanshub16** [comment]
Looks like most of the things are fine.
There are few more issues:

* You have just one commit to this PR. It becomes tough to figure out what exact changes were made after I last reviewed. Please don't edit commits, or merge them unless they are really the same and should be merged.

**Comment by himanshub16** [comment]
LGTM 👍 Kudos

**Pull request merged**